http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -49,6 +49,7 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift](
         handleStartRuntimeMessage(msg, now)
       case msg: UpdateRuntimeMessage =>
         handleUpdateRuntimeMessage(msg, now)
+      case _ => ??? //TODO: put disk messages here
```

Comment:
Cool, so handling this will be the work for https://broadworkbench.atlassian.net/browse/IA-1757

Reply:
can you add something like `throw new NotImplementedError("disk events are not implemented yet")`